modBoot/src/org/aion/Aion.java

```diff
@@ -71,34 +69,41 @@ public static void main(String args[]) throws InterruptedException {
          * if in the config.xml id is set as default [NODE-ID-PLACEHOLDER]
          * return true which means should save back to xml config
          */
-        if(cfg.fromXML())
-            cfg.toXML(new String[]{ "--id=" + cfg.getId() });
+        if (cfg.fromXML()) cfg.toXML(new String[] {"--id=" + cfg.getId()});
 
-        
         try {
             ServiceLoader.load(AionLoggerFactory.class);
         } catch (Exception e) {
             System.out.println("load AionLoggerFactory service fail!" + e.toString());
             throw e;
         }
 
-
-        // If commit this out, the config setting will be ignore. all log module been set to "INFO" Level
-        AionLoggerFactory.init(cfg.getLog().getModules());
+        // If commit this out, the config setting will be ignore. all log module been set to "INFO"
+        // Level
+        /** Changed INITIALIZE signature to include LOGFILE and LOGPATH */
+        AionLoggerFactory.init(
+                cfg.getLog().getModules(), cfg.getLog().getLogFile(), cfg.getLog().getLogPath());
         Logger LOG = AionLoggerFactory.getLogger(LogEnum.GEN.toString());
 
-        System.out.println(                
-                        "                     _____                  \n" +
-                        "      .'.       |  .~     ~.  |..          |\n" +
-                        "    .'   `.     | |         | |  ``..      |\n" +
-                        "  .''''''''`.   | |         | |      ``..  |\n" +
-                        ".'           `. |  `._____.'  |          ``|\n\n" +
-                        "                    NETWORK  v" + KERNEL_VERSION +
-                                "\n\n"
-                );
+        /** Outputs message to console depending on logPath validity */
+        if (cfg.getLog().isValidInput()) {
+            System.out.println("Logger file path: '" + cfg.getLog().getLogPath() + "'\n");
+        } else {
+            System.out.println("File path is invalid; set to default: 'log'\n");
+        }
```

Comment:
I think the better way is to stop the kernel instead of change to the default path.